src/patternfly/demos/Card/examples/CardView.md

```diff
@@ -284,3 +286,1028 @@ wrapperTag: div
+        {{#> description-list-group}}
+          {{#> description-list-term}}
+            OpenShift Version
+          {{/description-list-term}}
+          {{#> description-list-description}}
+            4.5.0.ci-2020-06-16-015028
+          {{/description-list-description}}
+        {{/description-list-group}}
+        {{#> description-list-group}}
+          {{#> description-list-term}}
+            Update Channel
+          {{/description-list-term}}
+          {{#> description-list-description}}
+            stable-4.5
+          {{/description-list-description}}
+        {{/description-list-group}}
+      {{/description-list}}
+    {{/card-body}}
+    {{> divider}}
+    {{#> card-footer}}
+      <a href="#">View Settings</a>
+    {{/card-footer}}
+  {{/card}}
+  {{#> card}}
+    {{#> card-title}}
+      {{#> title titleType="h2" title--modifier="pf-m-xl"}}
+        Details
+      {{/title}}
+    {{/card-title}}
+    {{#> card-body}}
+      {{#> description-list}}
+        {{#> description-list-group}}
+          {{#> description-list-term}}
+            Cluster API Address
+          {{/description-list-term}}
+          {{#> description-list-description}}
+            https://api2.devcluster.openshift.com
+          {{/description-list-description}}
+        {{/description-list-group}}
+        {{#> description-list-group}}
+          {{#> description-list-term}}
+            Cluster ID
+          {{/description-list-term}}
+          {{#> description-list-description}}
+            08908908-b850-41d9-8820-239becde9e86
+          {{/description-list-description}}
+        {{/description-list-group}}
+        {{#> description-list-group}}
+          {{#> description-list-term}}
+            Provider
+          {{/description-list-term}}
+          {{#> description-list-description}}
+            Azure
+          {{/description-list-description}}
+        {{/description-list-group}}
+        {{#> description-list-group}}
+          {{#> description-list-term}}
+            OpenShift Version
+          {{/description-list-term}}
+          {{#> description-list-description}}
+            4.5.0.ci-2020-06-16-015026
+          {{/description-list-description}}
+        {{/description-list-group}}
+        {{#> description-list-group}}
+          {{#> description-list-term}}
+            Update Channel
+          {{/description-list-term}}
+          {{#> description-list-description}}
+            stable-4.4
+          {{/description-list-description}}
+        {{/description-list-group}}
+      {{/description-list}}
+    {{/card-body}}
+    {{> divider}}
+    {{#> card-footer}}
+      <a href="#">View Settings</a>
+    {{/card-footer}}
+  {{/card}}
+{{/gallery}}
+```
+
+### Aggregiate status card
+```hbs
+{{#> grid grid--modifier="pf-m-gutter"}}
+  {{#> grid-item}}
+    {{#> gallery gallery--modifier="pf-m-gutter"}}
+      {{#> card}}
+        {{#> card-title card-title--modifier="pf-u-text-align-center"}}
```

Comment:
instead of center align on the card and body, could you just apply it to the parent card?